web-wallet/src/style/dusk/language.css

```diff
@@ -117,7 +117,7 @@
   /* Dividers */
 
   --divider-color-primary: var(--smokey-black);
-  --divider-background-color: var(--light-gray);
+  --divider-background-color: var(--light-grey);
```

Comment:
Nice catch, but in theory we should use American English in code, so we should change to "gray" in the future.

We live in _gray_ times.

Reply:
👀